cfgov/wellbeing/jinja2/wellbeing/results.html

```diff
@@ -582,21 +630,22 @@ <h4>Compare by</h4>
 
             <figcaption class="comparison-chart_source">
                 <dl>
-                    <dt>Source:</dt>
+                    <dt>{{ _('Source:') }}</dt>
                     <dd>
-                        <a href="/data-research/research-reports/financial-well-being-america/">Financial Well-Being in America report</a> |
-                        <a href="https://files.consumerfinance.gov/f/documents/cfpb_well-being_score-comparison-data.csv">Download CSV file</a>
+                        <a href="/data-research/research-reports/financial-well-being-america/">{{ _('Financial Well-Being in America report') }}</a> |
```

Comment:
This link text is not being translated.

Reply:
Oddly enough, that is the desired translation for the stakeholders.

<img width="507" alt="Screen Shot 2020-04-22 at 9 12 44 AM" src="https://user-images.githubusercontent.com/1183435/79986100-8675d580-8479-11ea-9df5-c5af1c7386ac.png">

Reply:
Hmm… I guess maybe because the report isn't translated? I might remove the translation wrapper, then, so future devs don't expect it to be translated when they see the code, and then it's not translated on-page.

Reply:
Addressed in newest commits.